dnf5/commands/download/download.cpp

```diff
@@ -273,7 +283,13 @@ void DownloadCommand::run() {
                 ctx.get_base().get_logger()->warning("Failed to get mirror for package: \"{}\"", pkg.get_name());
                 continue;
             }
-            std::cout << urls[0] << std::endl;
+            std::cout << urls[0];
+            if (urlallmirrors_option->get_value()) {
+                for (long unsigned index = 1; index < urls.size(); ++index) {
```

Comment:
Plese use "size_t" or "auto" instead of "long unsigned". The type should match a type returned by urls.size().